runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -648,8 +652,10 @@ class INTERPRETER_CLASS
 #endif /* J9SW_NEEDS_JIT_2_INTERP_CALLEE_ARG_POP */
 			/* Set the flag indicating that the caller was the JIT */
 			_currentThread->jitStackFrameFlags = J9_SSF_JIT_NATIVE_TRANSITION_FRAME;
-			/* If a stop request has been posted, handle it instead of running the native */
-			if (J9_ARE_ANY_BITS_SET(_currentThread->publicFlags, J9_PUBLIC_FLAGS_STOP)) {
+			if (isMethodDefaultConflictForMethodHandle) {
+				buildMethodFrame(REGISTER_ARGS, _sendMethod, jitStackFrameFlags(REGISTER_ARGS, 0));
```

Comment:
I don't think this belongs here - the frame is built in the default conflict code.